src/templates/layout.njk

```diff
@@ -1,69 +1,103 @@
 {% extends "govuk/template.njk" %}
-
-{# Import GOVUK components globally #}
 {% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
 
-{# Import custom components globally #}
-{% from "common/heading/macro.njk" import appHeading %}
-
-{% set containerClasses = "app-width-container--wide" %}
+{% set containerClasses = "" %}
 
 {% block head %}
   <link href="{{ getAssetPath('stylesheets/application.scss') }}" rel="stylesheet">
 {% endblock %}
 
-{% block header %}
-  {{ govukHeader({
+{%
+  set defaultHeaderOptions = {
     homepageUrl: "/",
-    classes: "app-header",
-    containerClasses: containerClasses,
     serviceName: serviceName,
     serviceUrl: serviceUrl,
     useTudorCrown: true
-  }) }}
-  {% endblock %}
+  }
+%}
+
+{%
+  set navigationOptions = {
+    navigationClasses: 'btms-header-nav',
+    navigation: [{
+      text: 'Manage account',
+      href: 'https://your-account.cpdev.cui.defra.gov.uk/management'
```

Comment:
this URL will be different in dev / ext-test / prod so it will need to be passed to the template

Reply:
In a future PR let's add an environment variable for the defra-id account management URL